dont-bother-looking.md

```diff
@@ -0,0 +1,49 @@
+## Title
+* Don't bother looking
+
+## Also Known As
+* Not looking for stuff internally 
+
+## Context
+* Software component(s) are available internally but users can't easily find these.
+* This problem is more likely to occur where there are silos in the company (e.g., larger companies; smaller companies may have fewer opportunities for reuse of internally developed software). 
+* The company traditionally has been bad at sharing across silos (people don't have the culture of sharing). 
+
+## Problem
+People don't bother looking for internally developed solutions - they might not find the repo at all or be aware of its existence.
+
+## Forces
+* No good internal search engine (or not connected to git repositories; and difficult to make this change happen)
+* Users may not know there are common places to find internally developed solutions.
+* People don't expect to find solutions internally.
+* Many silos in the company; difficult to reach the developer base across those silos (a communications problem).
+* People might not want to use internal SW because they don't believe it will be helpful
+    - might not be maintained
+    - might have poor reusability
+    - if someone put out a SW internally, the expectation is that they wouldn't have time to support it (vs. open source options)
+
+## Solution
+Make it easy to find the reusable code.
+
+* Pull in Repo names, descriptions and README.md files into the search engine
```

Comment:
What kind of search engine are we thinking of here?

Reply:
Probably whatever internal search engine people would use to try to find internally available software.

Reply:
That assumes that such a search engine exists.  It seems like some enterprise tooling has this kind of functionality built in, so the systems that a company is using may affect its ability to follow this part of the pattern.  In my experience systems like _GitHub_ and _npmjs_ try hard to include project search as a first-class feature.  I assume that their enterprise editions to the same, so companies using those tools would have an easier time implementing this advice.

Reply:
Even if *a* search engine exists internally, what I find really helpful is to have a one-stop-shop kind of search engine for all relevant communication and documentation. Even when using Github's enterprise offering, people often deploy additional systems like wikis to host content orthogonal to what is in the code repositories, slack channels (or IRC), mailing lists (or even nntp servers), some place to store stuff written down in office formats, search over personal e-mail etc. Several of these systems come with search built-in, but integrating this content in one search box or even just deploying a federated search engine across all sources often doesn't come off-the-shelf.

Reply:
Do you have examples of what a good search engine solution would be to aggregate many sources of information like that?  I've gotten the impression that it's not an easy problem.